src/scene-entry-manager.js

```diff
@@ -226,7 +226,11 @@ export default class SceneEntryManager {
   };
 
   _signInAndPinOrUnpinElement = (el, pin) => {
-    const action = pin ? () => this._pinElement(el) : async () => await this._unpinElement(el);
+    const action = pin
+      ? () => this._pinElement(el)
+      : async () => {
+          await this._unpinElement(el);
+        };
```

Comment:
This is a nit but I find that when these ternary operators become multi-line, you're better off just re-writing it to be not a ternary anymore, for the sake of readability

Reply:
ah sure. This was an accident of running `prettier` on this file -- not a change I thought about.